mods/ra/maps/allies-05a/map.yaml

```diff
@@ -1592,7 +1592,7 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		LuaScript:
-			Scripts: allies05a.lua, AI.lua
+			Scripts: allies05a.lua, AI05a.lua
```

Comment:
This becomes ambiguous if/when we have a soviet05a that also has an AI file.  I suggest `allies05a-ai.lua`.